### PR TITLE
 Space Travelers' Hub 

### PR DESCRIPTION
**Pull request by rcqx**
## Comments 
* Add Rockets, Missions, Profile, and Navigation components
* Setting dependencies 
* Use React documentation.
* Use React components.
* Use React props.
* Use React Router.
* Connect React and Redux.
* Handle events in a React app.
* Write unit tests with React Testing Library.
* Use styles in a React app.
* Use React life cycle methods.
* Apply React best practices and language style guides in code.
* Use store, actions and reducers in React.
* Perform a code review for a team member.

image

### CODE REVIEW

File: src/tests/missions.test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import store from '../redux/configureStore';
+import Missions from '../components/Missions';
+
+describe('Testing the Missions component', () => {
+  test('When data fetched from API all 10 missions are rendering succesfully', async () => {
+    render(<Provider store={store}><Missions /></Provider>);
+    await waitFor(() => {
+      expect(screen.getAllByText('Join mission').length).toBeGreaterThanOrEqual(10);
+    });
+  });
```
Comment (uwadonat) [inline src/tests/missions.test.js:14]: - [ ] Some tests are failing. Kindly fix this issue.
Screenshot from 2022-06-24 03-05-49

File: src/redux/rockets/api.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+// import { RESERVE_ROCKET, SET_ROCKETS } from './actiontypes';
+// import * as actions from './actiontypes';
```
Comment (uwadonat) [inline src/redux/rockets/api.js:2]: - [ ] remove these commented lines to make your code more clean

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi **@Team**

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your Pull request and commit messages are very descriptive ✔️
- No linter errors  ✔️
- Professional README file ✔️
- You used correctly gitflow ✔
almost
️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/missions.test.js (hunk 1 above)
Comment (Emmyn5600) [reply src/tests/missions.test.js:14]: This is not yet implemented as requested, please fix the error.

image

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @jr-cast 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- Check the comment under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @jr-cast and @AlphaNtihinduka    :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**